cdap-docs/tools/doc-cdap-default.py

```diff
@@ -101,6 +109,34 @@ def encode(value):
         else:
             return ''
 
+    @staticmethod
+    def _split_text(text, length=70, delimiter='.'):
+        # Splits text into blocks not exceeding length if possible
+        if len(text) > length and text.count(delimiter):
+            # Split text into multiple pieces
+            split = text.rfind('.', 0, length)
+            left = text[:split+1]
+            right = text[split+1:]
+            if right:
+                texts = Item._split_text(right, delimiter)
```

Comment:
This seems like improper usage of the function. It's passing `delimiter` in as the second parameter, but in the function definition, `delimiter` is the third parameter; `length` is the second parameter in the function definition.